src/kleinanzeigen_bot/resources/config_defaults.yaml

```diff
@@ -26,6 +26,12 @@ ad_defaults:
 #   Jobs > Praktika: 102/125
 categories: {}
 
+download:
+  # if true, all shipping options matching the size will be stored
```

Comment:
can you replace "size" with "package size"

Reply:
Thanks for the suggestion! I've updated the comment to be more precise about package size.